app/helpers/ui_helper.rb

```diff
@@ -0,0 +1,43 @@
+module UiHelper
```

Comment:
wouldn't any helper be a UI helper since these are usually methods used in views? Thoughts on a more specific name, such as `FormHelper` ?